src/main/java/net/glowstone/block/itemtype/ItemWearable.java

```diff
@@ -0,0 +1,60 @@
+package net.glowstone.block.itemtype;
+
+import org.bukkit.Material;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.inventory.ItemStack; 
+
+public class ItemWearable extends ItemType {
```

Comment:
Oh, sorry, did not realize that EnchantmentTarget included things like ARMOR_FEET, I just saw ARMOR.